src/ontology/edits/oeo-physical.omn

```diff
@@ -3213,6 +3221,120 @@ Class: OEO_00000449
         <http://purl.obolibrary.org/obo/RO_0000087> some OEO_00000001
     
     
+Class: OEO_00010000
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Ammonia is a portion of matter with the chemical formula NH3. It is a gas and can act as an air pollutant. Synonyms are trihydridonitrogen and nitrogen trihydride.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "NH3",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/263
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/356",
+        rdfs:label "ammonia"@de
+    
+    SubClassOf: 
+        OEO_00000331,
+        <http://purl.obolibrary.org/obo/RO_0000056> some OEO_00000055,
+        has_normal_state_of_matter value OEO_00000182
+    
+    
+Class: OEO_00010001
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Carbon monoxide is a portion of matter with the chemical formula CO. It is a gas and can act as an air pollutant.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "CO",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/263
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/356",
+        rdfs:label "carbon monoxide"@de
+    
+    SubClassOf: 
+        OEO_00000331,
+        <http://purl.obolibrary.org/obo/RO_0000056> some OEO_00000055,
+        has_normal_state_of_matter value OEO_00000182
+    
+    
+Class: OEO_00010002
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Nitrogen oxides are a portion of matter consisting of compounds of nitrogen and oxygen or a mixture of such compounds. It is a gas and can act as an air pollutant.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "NOx",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/263
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/356",
+        rdfs:label "nitrogen oxides"@de
+    
+    SubClassOf: 
+        OEO_00000331,
+        <http://purl.obolibrary.org/obo/RO_0000056> some OEO_00000055,
+        has_normal_state_of_matter value OEO_00000182
+    
+    
+Class: OEO_00010003
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Nitric oxide is a portion of matter with the chemical formula NO. It is a gas and can act as an air pollutant. Synonyms are nitrogen oxide or nitrogen monoxide.",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "Nitric oxide is a portion of matter with the chemical formula NO. It can act as an air pollutant. Synonyms are nitrogen oxide or nitrogen monoxide.",
```